src/shared_modules/content_manager/src/components/offlineDownloader.hpp

```diff
@@ -80,49 +83,98 @@ class OfflineDownloader final : public AbstractHandler<std::shared_ptr<UpdaterCo
     };
 
     /**
-     * @brief Downloads the requested local file and update the context accordingly.
+     * @brief Copy a file from the localsystem.
      *
-     * @note Despite the method name, there is no such download since the file is present in the filesystem.
+     * @param inputFilepath Input path from where to copy the file.
+     * @param outputFilepath Output path where to paste the file.
+     * @return true if the file was copied, otherwise false.
+     */
+    bool copyFile(const std::filesystem::path& inputFilepath, const std::filesystem::path& outputFilepath) const
+    {
+        constexpr auto FILE_PREFIX {"file://"};
+
+        // Remove file prefix.
+        auto unprefixedUrl {inputFilepath.string()};
+        Utils::replaceFirst(unprefixedUrl, FILE_PREFIX, "");
```

Comment:
Improve (naming): The unprefixedUrl start as a prefixedUrl, what about?
```suggestion
        // Remove file prefix.
        auto unprefixedUrl {inputFilepath.string()};
        const auto unprefixedUrl {Utils::leftTrim(inputFilepath.string(), FILE_PREFIX);
```

Tested [here](https://godbolt.org/z/vThfxbbx7)

Reply:
The `trimLeft` method deletes any char, contained in the `args` param, at the beginning, until a different char is present.

Look at [this](https://godbolt.org/z/fanq7EWzq), for example. It cuts a piece of the file name.

Reply:
Nothing to do then